SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1194,6 +1194,22 @@ void USpatialActorChannel::UpdateEntityACLToNewOwner()
 	}
 }
 
+bool USpatialActorChannel::ShouldChangeAuthorityIntent() const
+{
+	// The authority intent should be changed by this worker if the following conditions are satisfied:
```

Comment:
Could we invert these checks so we check if this worker is allowed, and the actor isn't locked, before checking if it needs to be updated?

Reply:
"IsLocked was intentionally afterwards since the ShouldRelinquishAuthority will typically be false (and exit early) whereas !IsLocked will nearly always be true and go onto the ShouldRelinquishAuthority anyway" -Ally 2020

Reply:
Will it? It's a virtual function call, with GridBasedLBS doing positional checking.